doc/releases/migration-guide-3.7.rst

```diff
@@ -331,6 +331,47 @@ Display
         };
     };
 
+* SSD16XX based displays now use the MIPI DBI driver class (:github:`73946`).
+  These displays must now be declared within a MIPI DBI driver wrapper device,
+  which will manage interfacing with the display. Note that the `dc-gpios` pin
```

Comment:
nit - non-blocking as this can be addressed as part of putting final touches to the migration guide later in the release cycle

```suggestion
  which will manage interfacing with the display. Note that the ``dc-gpios`` pin
```

Reply:
Fixed, need to clear the merge conflict anyways